app/Models/EntityLog.php

```diff
@@ -236,12 +240,15 @@ public function actions($action): array
      */
     public function scopeFilter(Builder $builder, HistoryRequest $request): Builder
     {
-        if ($request->filled('user')) {
+        if ($request->filled('user') && !$request->filled('action')) {
```

Comment:
This will become a mess if we add more filters (which we will ;-)) Please refactor so that the `orWhere` for each action is properly handled and doesn't cause copy-pasting https://stackoverflow.com/questions/51725715/orwhere-with-multiple-conditions-in-eloquent